packages/stencil/src/components.d.ts

```diff
@@ -254,9 +253,15 @@ export namespace Components {
         "value"?: string;
     }
     interface DatacomLi {
-        "variantItem": ItemStyle;
```

Comment:
Mind reverting the changes to this file? It's auto-generated.

Reply:
Fixed.